### PR TITLE
Tapping through drag targets.

### PR DESCRIPTION
**Pull request by Hixie**
Factor out the HitTestBehavior logic so that RenderMetaData can use it.

Use that in DragTarget.

### CODE REVIEW

**Comment by Hixie** [comment]
Fixes https://github.com/flutter/flutter/issues/1529

File: packages/flutter/lib/src/widgets/basic.dart [hunk 1]
```diff
@@ -2193,19 +2193,29 @@ class ExcludeSemantics extends OneChildRenderObjectWidget {
...
 
-  RenderMetaData createRenderObject() => new RenderMetaData(metaData: metaData);
+  RenderMetaData createRenderObject() => new RenderMetaData(
+    metaData: metaData,
+    behavior: behavior
+  );
 
   void updateRenderObject(RenderMetaData renderObject, MetaData oldWidget) {
     renderObject.metaData = metaData;
+    renderObject.behavior = behavior;
```
Comment (abarth) [inline packages/flutter/lib/src/widgets/basic.dart]: `..`

File: packages/flutter/lib/src/widgets/basic.dart [hunk 2]
```diff
@@ -2193,19 +2193,29 @@ class ExcludeSemantics extends OneChildRenderObjectWidget {
...
   }
 
   void debugFillDescription(List<String> description) {
     super.debugFillDescription(description);
+    description.add('behavior: $behavior');
     description.add('$metaData');
```
Comment (abarth) [inline packages/flutter/lib/src/widgets/basic.dart]: should we tag this `metadata:`

**Comment by abarth** [comment]
LGTM

**Pull request merged**